io/plugins/eu.esdihumboldt.hale.io.gml.ui/src/eu/esdihumboldt/hale/io/gml/ui/GmlReaderSettingsPage.java

```diff
@@ -70,6 +70,7 @@ public boolean updateConfiguration(StreamGmlReader provider) {
 				Value.of(requestPaginationEnabled.getSelection()));
 		provider.setParameter(StreamGmlReader.PARAM_FEATURES_PER_WFS_REQUEST,
 				Value.of(featuresPerRequest.getText()));
+		provider.setParameter(StreamGmlReader.PARAM_TMP_DIR_PATH, null);
```

Comment:
Why is this necessary? It should be an optional parameter, right?

Reply:
Yes, agreed.